### PR TITLE
Create naughtylist.pod

### PR DESCRIPTION
**Pull request by ktown007**

### CODE REVIEW

File: 2023/naughtylist.pod [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+=head2 todo.pl
+
+    !/usr/bin/env perl
+
+    use Dancer2;
+    set template => 'mustache';
+
+        #Initialize Santa's Naughty & Nice List
+
+    my $list = { list => [] };
```
Comment (briandfoy) [inline 2023/naughtylist.pod:21]: Maybe persist the list?

File: 2023/naughtylist.pod [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+
+=item 3.
+
+When you visit the root URL /, it serves an index.html file. Make sure to create this HTML file for a nice, festive interface.
+
+
+=item 4.
+
+To delete a naughty or nice kid, we look for the kid's task in the list and remove them. Then, we display the updated list using the 'list' template.
+
```
Comment (briandfoy) [inline 2023/naughtylist.pod:81]: Are both naughty and nice in the same list? I'd like to see how the app distinguishes between those two states. Maybe it's there and just not apparent to me.

File: 2023/naughtylist.pod (hunk 1 above)
Comment (ktown007) [reply 2023/naughtylist.pod:21]: I had the thought to do a part 2 using planetscale db and a part 3 with a fly.io deployment. Details of the final version are here:
https://github.com/ktown007/todoapp
if you think there is interest expanding on the array of hash to table of records concept could be done. "Santa's favorite Dancer Fly.io(s) to PlanetScale in time for Christmas"...

File: 2023/naughtylist.pod (hunk 2 above)
Comment (ktown007) [reply 2023/naughtylist.pod:81]: it is just a simple to do list with only the one field. The article added the Christmas theme and gave a purpose for the list :) so Timmy- nice and Jane - naughty etc works. I should do a screen shot like this.

File: 2023/naughtylist.pod (hunk 1 above)
Comment (ktown007) [reply 2023/naughtylist.pod:21]: This was also meant to contrast with all the other to do app tutorials for front end frameworks. In keeping it simple I was surprised how easy it was.

File: 2023/naughtylist.pod (hunk 2 above)
Comment (briandfoy) [reply 2023/naughtylist.pod:81]: That's something you should explain in the text, but, it wouldn't be hard to have two lists. :)

**Comment by oalders** [comment]
Closes #302

File: 2023/naughtylist.pod [hunk 3]
```diff
@@ -0,0 +1,196 @@
+Title: Santa's Christmas Dancer2 Web App - Naughty & Nice List Manager
+Topic: HMLX and Dancer2
```
Comment (oalders) [inline 2023/naughtylist.pod:2]:
```suggestion
Topic: HTMX and Dancer2
```

File: 2023/naughtylist.pod [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+=encoding utf8
+
+=pod 
+
+Ho, ho, ho! In this hilarious Christmas-themed Dancer2 web app tutorial, we'll help Santa manage his Naughty & Nice list. This time, we're using Perl's Dancer2 framework, but Santa's Christmas spirit is universal. Let's dive in and create Santa's very own web app!
+
+
+=head2 todo.pl
+
+    !/usr/bin/env perl
```
Comment (oalders) [inline 2023/naughtylist.pod:14]:
```suggestion
    !#/usr/bin/env perl
```

**Pull request merged**